apps/vscode/command_client/command_client.py

```diff
@@ -197,20 +197,33 @@ def run_command(
     return decoded_contents["returnValue"]
 
 
-def get_communication_dir_path():
-    """Returns directory that is used by command-server for communication
+command_server_directory = None
 
-    Returns:
-        Path: The path to the communication dir
-    """
-    suffix = ""
 
+# NB: See https://github.com/talonhub/community/issues/966 for why we do OS-specific temp dirs
+def get_platform_specific_communication_dir_path():
     # NB: We don't suffix on Windows, because the temp dir is user-specific
     # anyways
+    suffix = ""
     if hasattr(os, "getuid"):
         suffix = f"-{os.getuid()}"
```

Comment:
if we're not using this on windows, shall we put it inside the if statement where we actually use it?